### ISSUE TITLE
Comment concerning the bash scripts, python and giba 

### ISSUE THREAD

**Issue by heikef** [qmcurrents/gimic#181]
This comment is for users that find python scripts more convenient that bash scripts. 

My impression is when keeping things simple it should be straight forward to adapt the giba.py script for doing integration scans. 
Initially giba was written and shared by Raul Mera @rmera and it is intended to do scans related to the estimation of hydrogen bond strengths.

**Comment by mariavd** [comment]
I have never used them myself partly because I only knew that there is something by him but I was never told about using them. It is nice to be able to define the integration plane using command line arguments. It is a different approach from what I was doing and saves time in a different way. I think it will be worth writing about them in the documentation.

**Comment by heikef** [comment]
Perhaps some more python oriented users want to adapt them or add a new script? I have my own "poor man's version" because I wanted to practice python. It is not in a sharable stage. It is eventually for a new user also confusing to decide between python and bash scripts. Currently the bash ones are better maintained and offer more features.

**Comment by heikef** [comment]
Yes, we should add a comment into the documentation. Then it is clear that there is an option.

**Comment by heikef** [comment]
For the record: it turned out that giba.py is not strongly used and therefore not further tested. It's usage is at own risk.

**Comment by rmera** [comment]
I'd like to add that, even though Heike is correct, giba is a pretty small
program (~500 LOC iirrc), with limited use-cases. Of course I can't ensure
it to be bug-free, but I'd reckon it mostly is. If anyone would report a
bug to me, I would seriously try to understand my former self and fix it.

Cheers,

Raúl

**Comment by bast** [comment]
To be honest it is not clear to me why the issue was opened and then why it was closed? What was the issue?

**Comment by heikef** [comment]
Sorry, I take the blame for this confusion. 
1. I saw that giba.py could be adapted to work similar as the interactive bash scripts we have. It does not run interactive, but some user might prefer to work from the command line only. So I thought it is good to share this information. 
2. The present issue led to the conclusion that we could point out on giba.py in the manual. 
3. Then we had some general online / email exchange and it turned out that actually nobody (or only very limited persons) are using giba.py at all. 
In a sense there is no issue. Therefore I closed it.
